Userland/Libraries/LibJS/Runtime/ArrayBufferPrototype.cpp

```diff
@@ -128,6 +130,28 @@ JS_DEFINE_NATIVE_FUNCTION(ArrayBufferPrototype::slice)
     return new_array_buffer_object;
 }
 
+// 25.1.5.5 ArrayBuffer.prototype.transfer ( [ newLength ] ), https://tc39.es/proposal-arraybuffer-transfer/#sec-arraybuffer.prototype.transfer
```

Comment:
Please keep these in spec order - it was already a bit messy before, but now we have 3, 5, 6, 1, 4. Let's put these two after 4 and we can fix up the rest later.